pkgs/by-name/my/myanon/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, flex
+, bison
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "myanon";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "ppomes";
+    repo = "myanon";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-A4Qg8RvpkKgXoGTidshUyfX6FnhPR2ih1Nyne8Y87HA=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook flex bison ];
+
+  meta = with lib; {
+    description = "Myanon is a mysqldump anonymizer, reading a dump from stdin, and producing on the fly an anonymized version to stdout.";
+    mainProgram = "myanon";
+    homepage = "https://ppomes.github.io/myanon/";
+    platforms = platforms.unix;
+    license = licenses.bsd3;
+  };
```

Comment:
I my version I sorted the meta attributes, maybe it would be nice to keep them sorted.